cloud/docker_client.go

```diff
@@ -1,3 +1,4 @@
+//go:build go1.7
 // +build go1.7
```

Comment:
Do we need both of these lines?

Reply:
This is [the new syntax for build tags](https://pkg.go.dev/go/build@go1.17#hdr-Build_Constraints) as of go1.17, so `gofmt` auto-added it. It's not really a problem since go1.16 or below would just ignore this line. None of these tags should be necessary since we've been building with a version of go above 1.7 for a long while now, so I just removed all of them.